docs/background/object-storage.md

```diff
@@ -0,0 +1,63 @@
+# Object storage in {{brand}}
+
+Object storage in {{brand}} is implemented via the [Ceph Object Gateway](https://docs.ceph.com/en/latest/radosgw/index.html), also known as `radosgw` (commonly pronounced "[rados](https://ceph.io/assets/pdfs/weil-rados-pdsw07.pdf) gateway").
+
+This facility exposes access to objects via two RESTful APIs: the [Amazon S3 API](http://docs.aws.amazon.com/AmazonS3/latest/API/APIRest.html), and the [OpenStack Swift API](https://docs.openstack.org/api-ref/object-store/).
+It is, however, important to understand that radosgw shares no code with Amazon S3, nor with OpenStack Swift.
+
+As such, object storage in {{brand}} is not expected to behave *exactly* as Amazon S3 or OpenStack Swift, although it tracks their behavior very closely.
+The Ceph upstream documentation lists API deviations from the respective reference implementations, for both the [S3](https://docs.ceph.com/en/latest/radosgw/s3/) and the [Swift](https://docs.ceph.com/en/latest/radosgw/swift/) API.
+Our [reference section](../reference/index.md) lists additional [limitations](../reference/limitations/object-storage.md) specific to {{brand}}.
+
+
+## Object storage integration with OpenStack authentication
+
+In {{brand}}, the OpenStack `object‑store` service endpoints point not to native OpenStack Swift proxy servers, but to radosgw endpoints.
+This is to say that radosgw acts as a "drop-in" replacement for OpenStack Swift.
+
+As such, it is fully integrated with OpenStack's authentication facility, OpenStack Keystone.
+This means that once you have configured the [OpenStack CLI](../howto/getting-started/enable-openstack-cli.md) correctly, you are able to interact with the `object‑store` endpoint as with any other OpenStack service, using the Swift API.
+
+The S3 API, however, does not natively "know" about OpenStack authentication.
```

Comment:
OK, this is more of a personal preference: When writing something metaphorical, I tend to omit double quotes around words or sentences. In that particular case, by skipping the double quotes around "know" (oops), I would say to the reader something along the lines of "I know you know the S3 API cannot know anything, for it is **not** sentient".

Reply:
I see your point, but I would still prefer to leave the scare quotes in. :)